scripts/exgfs_atmos_gempak_meta.sh

```diff
@@ -62,7 +71,22 @@ do
       rm $DATA/poescript
 #   fi
 
-   awk '{print $1}' $FIXgempak/gfs_meta > $DATA/tmpscript
+   if [ $fhr -lt 100 ] ; then
+      typeset -Z2 fhr
```

Comment:
1) This does not need to be in an if-block. If fhr > 100, it will simply do nothing.
2) This is a functional change from before, which zero-padded to three places. This is only zero-padding to two
3) If the if-block mattered and we kept the padding as-is, zero-padding would only be needed < 10.

Reply:
Will update in `develop` branch.